### PR TITLE
LPS-174153 [Raylife AP] - Visual Adjustments

### PR DESCRIPTION
**Pull request by eduardo-diniz**
[LPS-174153](https://issues.liferay.com/browse/LPS-174153)

### CODE REVIEW

**Comment by eduardo-diniz** [comment]
ci:test:sf

**Comment by eduardo-diniz** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/Activities/TableListMobile.tsx [hunk 1]
```diff
@@ -120,7 +124,7 @@ const TableListMobile: React.FC<Props> = ({headers, rows}) => {
 														isPanelExpanded[
 															index
 														] === false,
-													'mb-2 pt-0':
+													'mb-2 pt-0 ':
```
Comment (claricebrasil) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/Activities/TableListMobile.tsx:127]:
```suggestion
													'mb-2 pt-0':
```

**Review by claricebrasil — CHANGES_REQUESTED** [review]

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/Activities/index.scss [hunk 2]
```diff
@@ -1,135 +1,153 @@
...
 	}
-	thead {
-		display: table;
-		width: 100%;
+
+	.bg-message {
+		background-color: #fff;
+		border-bottom-left-radius: 0.25rem;
+		border-bottom-right-radius: 0.25rem;
 	}
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/Activities/index.scss:68]: Try to use bootstrap Borders and dialect colors here.

**Review by joanagrego — CHANGES_REQUESTED** [review]
Good job! I added some suggestions :)

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/Activities/index.scss (hunk 2 above)
Comment (eduardo-diniz) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/Activities/index.scss]: Done. THx!

**Comment by eduardo-diniz** [comment]
ci:test:sf

**Comment by eduardo-diniz** [comment]
ci:test:sf

**Comment by claricebrasil** [comment]
ci:forward

**Pull request closed without merging**